pkgs/development/tools/aws-sam-cli/default.nix

```diff
@@ -13,6 +13,15 @@ let
         };
       });
 
+      requests = super.requests.overridePythonAttrs (oldAttrs: rec {
+        version = "2.22.0";
```

Comment:
Don't we have this in nixpkgs?
```
 nix eval -f . python3.pkgs.requests.version
"2.22.0"
```

Reply:
currently, yes

Reply:
must have been a recent change, it was 2.21 when i wrote the pr